sdk/appconfiguration/app-configuration/samples/javascript/getSettingOnlyIfChanged.js

```diff
@@ -14,7 +14,7 @@ async function main() {
   console.log("Running get setting only if changed sample");
 
   // Set the following environment variable or edit the value on the following line.
-  const connectionString = process.env["APPCONFIG_CONNECTION_STRING"] || "<connection string>";
+  const connectionString = process.env["AZ_CONFIG_CONNECTION"] || "<connection string>";
```

Comment:
@willmtemple -- Modified some variables to match those exported by the `test-resources.json` ARM template.

Reply:
CC @richardpark-msft

Reply:
In this case I think the name of the var in `test-resources.json` and elsewhere in the tests should be changed to match `APPCONFIG_CONNECTION_STRING`, since it follows the pattern laid out in other packages e.g. `TEXT_ANALYTICS_API_KEY`.

Also, like with keyvault, there is associated documentation on the APPCONFIG_CONNECTION_STRING variable name in sample.env that would need to be changed as well.